fuzz/fuzz_targets/no-traps.rs

```diff
@@ -0,0 +1,68 @@
+#![no_main]
+
+use arbitrary::Unstructured;
+use libfuzzer_sys::fuzz_target;
+use wasmtime::*;
+
+#[cfg(feature = "wasmtime")]
+#[path = "../../crates/fuzz-stats/src/dummy.rs"]
+pub mod dummy;
+
+// Define a fuzz target that accepts arbitrary
+// `Module`s as input.
+fuzz_target!(|data: &[u8]| {
+    let mut u = Unstructured::new(data);
+    let (wasm_bytes, config) = match wasm_tools_fuzz::generate_valid_module(&mut u, |config, _| {
+        config.disallow_traps = true;
+        config.threads_enabled = false;
+        config.allow_start_export = false;
+        config.max_memories = config.max_memories.min(1);
+        config.memory64_enabled = true;
+        config.exceptions_enabled = false;
```

Comment:
These config options were mostly an attempt to get around errors that were preventing the Module or Store from getting instantiated. I would not be surprised if some of these aren't actually necessary.